manifests/minio-operator-rhmp.clusterserviceversion.yaml

```
@@ -690,7 +690,7 @@ spec:
                     env:
                       - name: CONSOLE_OPERATOR_MODE
                         value: "on"
-                    image: quay.io/minio/console@sha256:8dce83e096fb72373aa27634f5d9d0cad574d6d89dd4d89f9b27334d77e7465e
+                    image: quay.io/minio/console@
```

```suggestion
                    image: quay.io/minio/console@
```

This looks wrong @dvaldivia

---

looking into it @harshavardhana , any idea what could be causing this @cniackz ?

---

it's fixed now, some bad yaml is getting removed as well